docs/get-started/v2/get-started/prebuilt/prebuilt.mdx

```diff
@@ -75,26 +78,27 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
     src="https://<your-template-subdomain>/meeting/<room-code>"
     style="height: 100vh; width: 100%; border:0;"
 >
+
+
 </iframe>
 ```
 
 </StepsContainer>
 
 ## Embed as Component
 
-100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing HMSRoomKit in your apps.
+100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing `RoomKit` package in your app.
```

Comment:
this needs more context. we're introducing a major thing here but the copy is not doing justice to it. need to elaborate on which platforms this is available, a line on what customisation is possible?